Sample-01/src/index.js

```diff
@@ -15,14 +15,16 @@ const onRedirectCallback = (appState) => {
 
 // Please see https://auth0.github.io/auth0-react/interfaces/auth0provideroptions.html
 // for a full list of the available properties on the provider
+const providerConfig = {
+  domain: config.domain,
+  clientId: config.clientId,
+  ...{ audience: config.audience },
```

Comment:
Same here, I am not sure I understand why we are spreading.

Reply:
Same as above - if `config.audience` is null, this prevents it from being merged into the object.

Reply:
I don't think that is true: https://jsfiddle.net/grqozm4h/

Reply:
I think you want `...(config.audience ? { audience: config.audience } : null),`